src/lib/spd_cache.c

```diff
@@ -209,7 +209,9 @@ enum cb_err spd_fill_from_cache(uint8_t *spd_cache, struct spd_block *blk)
 
 	dram_type = *(spd_cache + SC_SPD_OFFSET(i) + SPD_DRAM_TYPE);
 
-	if (dram_type == SPD_DRAM_DDR4)
+	if (dram_type == SPD_DRAM_DDR5)
+		blk->len = CONFIG_DIMM_SPD_SIZE;
```

Comment:
Shouldn't it be something like SPD_PAGE_LEN_DDR5?

Reply:
indeed, done in ba0140a9076a77760d1cb5e0a94db6e939abe726

Reply:
I disagree on the naming (but not on the fact that it shouldn't be `CONFIG_DIMM_SPD_SIZE`), now `MAX_SPD_PAGE_SIZE_SPD5` < `SPD_PAGE_LEN_DDR5`.

Reply:
Renamed to SPD_LEN_DDR5 in https://github.com/Dasharo/coreboot/compare/ba0140a9076a77760d1cb5e0a94db6e939abe726..bd6336a138437d57fdc4c2097a664150e417d83f